careers/ladder.md

```diff
@@ -563,6 +578,65 @@ See
 [this document](https://www.notion.so/artsy/Artsy-Tech-Lead-Expanded-Responsibilities-Public-710ee0cd2fce47dcaa722ece80e365b2)
 for an expanded set of Tech Lead Responsibilities at Artsy.
 
+## Staff Engineer Archetypes
+
+### Optimizers / Fixers / Solver
+
+These engineers love to dive deep into a complex problem and resurface with a solution that is accessible to team
```

Comment:
Would also add here that this archetype tends to go wherever they're needed, and might end up switching teams often in order to follow complex, impactful problems.